models/iaf_cond_alpha_mc_fixedca.h

```diff
@@ -0,0 +1,663 @@
+
+There is one excitatory and one inhibitory conductance-based synapse
+onto each compartment, with alpha-function time course. The alpha
+function is normalised such that an event of weight 1.0 results in a
+peak current of 1 nS at t = tau_syn. Each compartment can also receive
+current input from a current generator, and an external (rheobase)
+current can be set for each compartment.
+
+Synapses, including those for injection external currents, are addressed through
+the receptor types given in the receptor_types entry of the state dictionary. Note
+that in contrast to the single-compartment iaf_cond_alpha model, all synaptic
+weights must be positive numbers!
+
+Depending on value of active flag, calcium spike maybe triggered upon threshold crossing,
+V_m.d >= V_thCa. The spike is modeled using a fixed waveform, a current array with fixed length.
+The duration of the calcium spike is of same length as its refractory period, hence no new
+calcium spike maybe triggered while a calcium spike is still active.
+
+Parameters:
+The following parameters can be set in the status dictionary. Parameters
+for each compartment are collected in a sub-dictionary; these sub-dictionaries
+are called "soma", "proximal", and "distal", respectively. In the list below,
+these parameters are marked with an asterisk.
+
+V_m*         double - Membrane potential in mV
+E_L*         double - Leak reversal potential in mV.
+C_m*         double - Capacity of the membrane in pF
+E_ex*        double - Excitatory reversal potential in mV.
+E_in*        double - Inhibitory reversal potential in mV.
+g_L*         double - Leak conductance in nS;
+tau_syn_ex*  double - Rise time of the excitatory synaptic alpha function in ms.
+tau_syn_in*  double - Rise time of the inhibitory synaptic alpha function in ms.
+I_e*         double - Constant input current in pA.
+t_L*         double - Leak during refractory period in nS
+nt_L*        double - Leak at other times in ns
+tau_curAP*   double - Time constant of active current at each compartment after action potential in
+pA
+amp_curAP*   double - Amplitude of active current at each compartment after action potential in pA
+
+
+g_sp         double - Conductance connecting soma and proximal dendrite, in nS.
+g_pd         double - Conductance connecting proximal and distal dendrite, in nS.
+t_ref        double - Duration of refractory period in ms.
+V_th         double - Spike threshold in mV.
+V_reset      double - Reset potential of the membrane in mV.
+V_max        double - Peak voltage at spike in mV
+reset_flag   double - flag to set somatic membrane potential to reset value at end of refractory
+period
+jump_Th      double - Jump in adaptive threshold upon spike in mV
+tau_Th       double - Time constant for adaptive threshold in ms
+V_thCa       double - Threshold for calcium spike in mV
+amp          double - Amplitude factor for calcium spike current
+Ca_amp [CA_SIZE]  double - Waveform for calcium spike current in pA
```

Comment:
Is `Ca_amp` a vector of doubles, one value per time step? If, as described above, `t_refCa` is equal to the length of the waveform, why do you have it as as separate variable? Doesn't it follow directly from the length of `Ca_amp` and the time resolution?

I am also not sure if I like the `amp` and `Ca_amp` parameter names. I think `amp` should be `Ca_amplitude` and `Ca_amp` should be `Ca_waveform`. I would then assume the waveform to have a maximal amplitude of 1, so that the `Ca_amplitude` would give the actual amplitude. I think it makes more sense then to assign the unit to `Ca_amplitude` and consider the waveform dimensionless.